scripts/test_write_input.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import write_input as wi
+import numpy as np
+import collections
+import os
+
+dir = os.path.dirname(__file__)
+test_database_path = os.path.join(dir, 'test_database.csv')
+
+class WriteInputTest(unittest.TestCase):
+    """ Tests for write_input.py """
+
+    def test_read_csv(self):
+        """ Test if read_csv reads the csv correctly"""
```

Comment:
This docstring doesn't communicate anything useful. What is "correctly" in this case? Also, remove the space after the first triple-quote.